p2p_wallet/Extensions/Double+Extensions.swift

```diff
@@ -83,7 +83,7 @@ extension Double {
         showMinus: Bool = true,
         groupingSeparator: String? = nil,
         autoSetMaximumFractionDigits: Bool = false,
-        roundingMode: NumberFormatter.RoundingMode? = nil
+        roundingMode: NumberFormatter.RoundingMode? = .down
```

Comment:
Changing default property is dangerous, don't you think that?

Reply:
Yes I totally agree. However, I guess it is right solution for money rounding because we can't show users bigger amount than they have. I believe all money amounts show be rounded down and it should be tested. Here is Alexey suggestion : https://discord.com/channels/767761912167006292/1095338026642710648/1095340220351795200. I don't see any possible vulnerabilities if we have rounding only in string formats.  Previously, we had this setting only for send screen, it might be overwhelming to set this setting everywhere in the methods across the application. Should I ask Nick also?

Reply:
Do you still think we need to put roundingMode down for separate amounts?

Reply:
I think we should ask the team for reviewing this issue in case it breaks someone's work. Added @sidorov-panda @qwerkly as reviewer

Reply:
why did we remove optional default value?